dev_ws/src/operations/safety_node/include/safety_node.h

```diff
@@ -0,0 +1,27 @@
+#ifndef SAFETY_NODE_H_
```

Comment:
google style guide says to use header guards, and .h. I agree it's probably easier to use #pragma once, but to make things easier on leads and future members should we just keep it exactly to google style guide so that if a new member asks, leads can just say go look at the style guide cause we follow it bar for bar?